README.md

```
@@ -127,38 +143,30 @@ jobs:
 
+## 🔐 PRs from Forked Repositories
 
-### Usage to deal with PRs created from forked repositories
-
-When someone creates a PR from a forked repository, there is a security challenge: workflows triggered by `pull_request` events do not have access to your to repository secrets (like your surge token) for security reasons.
+When someone creates a PR from a forked repository, there is a security challenge: workflows triggered by `pull_request` events do not have access to your repository secrets (like your surge token) for security reasons.
 
-**Why this is a problem:** Without access to the surge token, the preview deployment will fail.
+> [!WARNING]
+> **Why this is a problem:** Without access to the surge token, the preview deployment will fail.
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)